root/about/sponsors.tx

```diff
@@ -319,7 +319,23 @@ community.
                 </td>
             </tr>
             <tr>
-                <td><a href="https://kritika.io"><img src="/static/images/sponsors/kritika.svg" class="image-responsive"></a></td>
+                <td><a href="https://deriv.com/careers/"><img src="/static/images/sponsors/deriv-horizontal-web-dark-100.jpeg" width="170" align="left"></a></td>
```

Comment:
Could we add `target="_blank" rel="noopener"` to all of the `href` tags?

Reply:
Done :+1: